src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUtils.java

```diff
@@ -39,6 +48,30 @@
   /** The designator for the local time zone. (i.e. -05:00, +02:00, Z, etc.) */
   private static String localTimeZoneDesignator = null;
 
+  /**
+   * String value indicating if tag data is exported using UTC time stamps. No value is used to
+   * indicate that the actual value has not been read yet. The values 0 and 1 are used to indicate
+   * false and true respectively.
+   */
+  private static String tagDataExportedInUtc = "";
+
+  /**
+   * Gets a boolean value indicating if tag data is exported using UTC time stamps.
+   *
+   * @return true if tag data is exported using UTC time stamps, false otherwise
+   * @throws Exception if unable to get UTC export value
+   */
+  public static boolean getTagDataExportedInUtc() throws Exception {
+    if (tagDataExportedInUtc.length() == 0) {
+      // Create system control block
+      SysControlBlock sysControlBlock = new SysControlBlock(SysControlBlock.SYS);
+
+      // Get UTC export value from system control block
+      tagDataExportedInUtc = sysControlBlock.getItem(SCB_UTC_EXPORT_KEY).trim();
+    }
+    return tagDataExportedInUtc.equals(SCB_UTC_EXPORT_VALUE_TRUE);
```

Comment:
how many times this function is called?  It may be more performant to change localTimeZoneDesigner to boolean and return this ( if it is not null). It would be quicker than doing a string comparison each time the function is called.

Reply:
Booleans cannot be null, although I think this is a valid point. I will address so that a simple value is returned instead of a calculation each time.

Reply:
Addressed